src/test/suite/fixtures/recorded/actions/clearThat.yml

```diff
@@ -0,0 +1,29 @@
+languageId: plaintext
```

Comment:
Surely we already have a test for the "that" mark? If not then I'd move this one into `marks` like the "source" test

Reply:
comment about source